src/App.vue

```diff
@@ -283,6 +325,9 @@ export default defineComponent({
 
       // load initial UC component state
       this.$store.dispatch(`zap/loadUcComponentState`)
+      if (this.uiThemeCategory == 'matter' && query[`newConfig`]) {
+        this.loadRootNode()
```

Comment:
Don't "load root node". UI should always just ask backed: "What should I do?"

Reply:
fixed, thank you